ThePortugueseMan.HabitTracker/Habit_DataBase/DataBaseCommands.cs

```diff
@@ -0,0 +1,394 @@
+﻿namespace DataBaseLibrary;
+using Microsoft.Data.Sqlite;
+using System;
+using System.Globalization;
+
+
+public class DataBaseCommands
+{
+    static string connectionString = @"Data Source=habit-Tracker.db";
+    static string s_MainTableName = "HabitsTable";
+
+    //if the main table doesn't exist, it's created
```

Comment:
Please check the necessity of this comment. 😊

Reply:
Made the function name more clear and removed the comment